helix-term/src/commands.rs

```
@@ -2485,6 +2485,10 @@ mod cmd {
         args: &[&str],
         _event: PromptEvent,
     ) -> anyhow::Result<()> {
+        if args.len() == 0 {
+            return Ok(());
```

We should return an error here, rather than failing silently.
```suggestion
            bail!("Line number required");
```

---

Thanks, that would be better than silencing the false path. :)

---

@Omnikar Fixed. :)